staging/generic-ingress/templates/ingresses.yaml

```
@@ -0,0 +1,20 @@
+{{- range $ingress := .Values.ingresses }}
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $ingress.name }}
+  labels:
+{{ include "generic-ingress.labels" $ | indent 4 }}
```

To be consistent with using nindent in file:

```suggestion
    {{ include "generic-ingress.labels" $ | nindent 4 }}
```